test/support/fake_handler.ex

```diff
@@ -1,3 +1,151 @@
 defmodule FakeHandler do
```

Comment:
Other helper module was scoped to Janus.ConnectionTest. My idea would be to scope both of these to `Janus.HandlerTest.XXX`